charts/hedera-network/templates/services/network-node-svc.yaml

```
@@ -4,6 +4,8 @@ apiVersion: v1
 kind: Service
 metadata:
   name: network-{{ $nodeConfig.name }}-svc
+  labels:
+    fullstack.hedera.com/type: network-node-svc
```

We should probably add a node name label here for use with label selector specs.

```suggestion
    fullstack.hedera.com/type: network-node-svc
    fullstack.hedera.com/node-name: {{ $nodeConfig.name }} 
```